pkg/pmem-csi-driver/controllerserver-master.go

```diff
@@ -4,7 +4,7 @@ Copyright 2017 The Kubernetes Authors.
 SPDX-License-Identifier: Apache-2.0
 */
 
-package pmemcsidriver
+package csidriver
```

Comment:
This is the only change that concerns me: while some of the code in  pkg/pmem-csi-driver is generic, not everything is. Therefore "pmemcsidriver" is the better package name.

Ultimately, we would need a `pkg/csi-driver` with the generic code and `pkg/pmem-csi-driver` with the PMEM-CSI specific code. But that is a larger change where I am not sure whether it is justified.

Can we leave the package name as it is for now?

Reply:
I though that the idea is to make it generic. Otherwise I'd need to copy most of its code to a new driver.

Reply:
In the new driver, do you want to work with a fork of the code or import he code from the PMEM-CSI repo?

My assumption was that we clean up the PMEM-CSI driver code so that it better distinguishes between generic code and PMEM code. That's worthwhile by itself.

But I thought you would still maintain a full copy of the code with additional modifications. In that model, a generic "pkg/csi-driver" is nice, but not required.

Reply:
My idea was to generalise driver code to the point it can be used(imported?) by other drivers.
However, if you just want a names cleanup it's even better as it's much less work for me :)
I've updated the PR as you suggested.